perceval/backends/jira.py

```diff
@@ -163,6 +167,34 @@ def parse_issues(raw_page):
         for issue in issues:
             yield issue
 
+    @staticmethod
```

Comment:
Understood! :relaxed: One last thing, as I'm moving this functions outside the `Backend`, I guess I must add a test for `map_custom_fields` (as it won't be private), but this testing is already implicit in `test_fetch`. Does it makes sense to add another test? (in terms of coverage).